### PR TITLE
Arrays ar dr

### PR DESCRIPTION
**Pull request by Rethora**
Alex and I have finished the array challenges.

### CODE REVIEW

File: arrays-alex-dane.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+
+
+var groceryList = ["chips", "dip", "cookies"]
+
+// Write the code that will add "soda" to the end of the original array.
+console.log(groceryList.push("soda"))
+
+// Write the code that will add "granola" to the end of array without altering the original array.
+var groceryList = ["chips", "dip", "cookies"]
```
Comment (SunkissedQueen) [inline arrays-alex-dane.js:9]: All of this work you did is valid. The challenge questions from line 5 through line 22 are to use the original array 
var groceryList = ["chips", "dip", "cookies"]
By reassigning a value, it changes the expected output for the subsequent questions. You definitely have the concept and workflow of arrays. Try commenting out line 9 and then use the rest of your code. You will notice the difference of what mutators and accessors do to an array.

File: arrays-alex-dane.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+// Write the code that will add "beans" to the "chips" and "dip" array.
+
+var slicedarr = groceryList.slice(0, 2)
+console.log(slicedarr.push("beans"))
+console.log(slicedarr)
+
+// Consider the variable:
+
+var numbers = [2, 4, 6, 8, 10]
```
Comment (SunkissedQueen) [inline arrays-alex-dane.js:26]: Similar recommendation here. Comment out line 32, 37, 42

File: arrays-alex-dane.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+var sortedNames = arr1.concat(arr2).sort()
+console.log(sortedNames)
+
+// Consider the variables:
+
+var numbers = [42, 221, 71, 7, 18, 87]
+var oddIndexes = []
+// Write the code that logs the values from the numbers array that are at odd indexes.
+
+for (let i = 0; i < numbers.length; i++) {
```
Comment (SunkissedQueen) [inline arrays-alex-dane.js:111]: This for loop is good for finding an odd value. To look for the odd index, start the index at the first odd index, change the incrementation to go to the next odd index. For instance (indentations are not reflected in this message line)
`for (let i = 1; i < numbers.length; i += 2) {`
    `var current = numbers[i]`
    `console.log(current)`
`}`

File: arrays-alex-dane.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    var current = numbers[i]
+    if (current % 2 !== 0) {
+        console.log(current)
+    }
+}
+
+// Write the code that adds the values from odd indexes into the oddIndexes array.
```
Comment (SunkissedQueen) [inline arrays-alex-dane.js:118]: Similar from above. Refactor this for loop to begin at the first odd index and increment to the next odd index. Your push method is spot on!

**Review by SunkissedQueen — APPROVED** [review]
Your hard work is shining through. If you have any questions on my comments, please reach out.

File: arrays-alex-dane.js (hunk 3 above)
Comment (Rethora) [reply arrays-alex-dane.js:111]: I sure have to get better at slowing down my reading and understanding the question word for word. Thanks for the feedback!

**Pull request merged**